### PR TITLE
Fix 'suppress' style warnings

### PR DESCRIPTION
**Pull request by blairconrad**
Cleans up style warnings revealed after conversion to StyleCop.Analyzers in #1090.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Try to keep lines of code no longer than 160 characters wide. This isn't a stric
 
 ## Coding Style
 
-Try to keep your coding style in line with the existing code. It might not exactly match your preferred style but it's better to keep things consistent. StyleCop compliance is enforced through analysis on each build and the treatment of warnings as errors. Any StyleCop settings changes or suppressions must be clearly justified.
+Try to keep your coding style in line with the existing code. It might not exactly match your preferred style but it's better to keep things consistent. Coding style compliance is enforced through analysis on each build. Any StyleCop.Analyzers settings changes or suppressions must be clearly justified.
```
Comment (thomaslevesque) [inline CONTRIBUTING.md]: 👍 
I wouldn't have thought about this.

File: src/FakeItEasy/Core/FakeManager.cs [hunk 2]
```diff
@@ -125,8 +125,7 @@ public void AddInterceptionListener(IInterceptionListener listener)
             this.interceptionListeners.AddFirst(listener);
         }
 
-        [SuppressMessage("Microsoft.Design", "CA1033:InterfaceMethodsShouldBeCallableByChildTypes",
-            Justification = "Explicit implementation to be able to make the IFakeCallProcessor interface internal.")]
+        [SuppressMessage("Microsoft.Design", "CA1033:InterfaceMethodsShouldBeCallableByChildTypes", Justification = "Explicit implementation to be able to make the IFakeCallProcessor interface internal.")]
```
Comment (thomaslevesque) [inline src/FakeItEasy/Core/FakeManager.cs]: I hesitated between putting everything on one very long line and putting each parameter on its own line... But I guess it's less intrusive this way.

File: CONTRIBUTING.md (hunk 1 above)
Comment (blairconrad) [inline CONTRIBUTING.md]: I'm all about those docs. ;)

**Comment by blairconrad** [comment]
Thanks for the merge!

**Comment by blairconrad** [comment]
This change has been released as part of [FakeItEasy 3.4.0](https://github.com/FakeItEasy/FakeItEasy/releases/tag/3.4.0).

**Pull request merged**